python/pyspark/tests.py

```diff
@@ -740,6 +739,27 @@ def test_multiple_python_java_RDD_conversions(self):
         converted_rdd = RDD(data_python_rdd, self.sc)
         self.assertEqual(2, converted_rdd.count())
 
+    def test_narrow_dependency_in_join(self):
+        rdd = self.sc.parallelize(range(10)).map(lambda x: (x, x))
```

Comment:
do these tests actually check for a narrow dependency at all?  I think they will pass even without it.

I'm not sure of a better suggestion, though.  I had to use `getNarrowDependencies` in another PR to check this:
https://github.com/apache/spark/pull/4449/files#diff-4bc3643ce90b54113cad7104f91a075bR582

but I don't think that is even exposed in pyspark ...